### PR TITLE
tests: Re add scanner_test.go

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

**Comment by simar7** [comment]
@nikpivkin can you take a look?

File: pkg/scanner/scanner_test.go [hunk 1]
```diff
@@ -0,0 +1,534 @@
+package scanner
+
+import (
+	"context"
+	"io/fs"
+	"testing"
+
+	"github.com/aquasecurity/defsec/pkg/providers/azure"
```
Comment (nikpivkin) [inline pkg/scanner/scanner_test.go:8]: We should not have a dependency on azure.

File: pkg/scanner/options.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package scanner
 
 import (
-	"github.com/aquasecurity/defsec/pkg/scanners/options"
+	"github.com/aquasecurity/trivy-iac/pkg/scanners/options"
```
Comment (nikpivkin) [inline pkg/scanner/options.go:4]: Should we have trivy-iac as a dependency?

File: pkg/scanner/scanner_test.go (hunk 1 above)
Comment (simar7) [reply pkg/scanner/scanner_test.go:8]: yes but it's part of a multi cloud test that today lives here https://github.com/aquasecurity/trivy-aws/blob/bc16385b39ea55d6a4d789730c91d41849576d8f/pkg/scanner/scanner_test.go#L358

File: pkg/scanner/options.go (hunk 2 above)
Comment (simar7) [reply pkg/scanner/options.go:4]: hmm yeah maybe the `options` package needs to be abstracted out of `trivy-iac` as well to a common place. What do you think?

Comment (simar7) [reply pkg/scanner/options.go]: I updated it here https://github.com/aquasecurity/trivy-aws/pull/8/commits/ba16ec1c71e6bd709773ee67047cd2d5ddc2d75b

**Review by nikpivkin — APPROVED** [review]
LGTM

**Pull request merged**